source/_integrations/universal.markdown

```diff
@@ -12,7 +12,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-A Universal Media Player can combine multiple existing entities in Home Assistant into a single media player entity. This is used to create a single media player entity that can control an entire media center.
+A Universal Media player can combine multiple existing entities in Home Assistant into a single media player entity. This is used to create a single media player entity that can control an entire media center.
```

Comment:
```suggestion
A Universal Media Player can combine multiple existing entities in Home Assistant into a single media player entity. This is used to create a single media player entity that can control an entire media center.
```
I'm not entirely sure about this one. It looks like [Universal Media Server](https://www.universalmediaserver.com/) is the name of their product. I did not find the player on their website. Maybe we can leave the caps, to align with the spelling of their product name.